### PR TITLE
indexer: add cli to add aliases without prefix if prefix is set

### PR DESCRIPTION
**Pull request by pazembrz**
*INSPIR-3504

### CODE REVIEW

**Comment by drjova** [comment]
@pazembrz  After our IRL discussion, we need to add a param to specify the prefix for the aliases name, for example:
```
--prefix-alias "inspire-"
```

will generate the following aliases:

* ```inspire-records-hep```
* ```inspire-records-authors```
* ...

File: backend/tests/integration/indexer/test_cli.py [hunk 1]
```diff
@@ -192,3 +192,82 @@ def test_remap_index_when_there_are_more_than_one_indexes_with_same_name_but_dif
...
+    assert "records-authors" in aliases
+    assert "records-seminars" in aliases
+    assert "records-jobs" in aliases
+    assert "records-institutions" in aliases
+    assert "records-journals" in aliases
+    assert "records-data" in aliases
+    assert "records-experiments" in aliases
+    assert "records-conferences" in aliases
+
+    # cleanup
```
Comment (drjova) [inline backend/tests/integration/indexer/test_cli.py:223]: redundant

File: backend/tests/integration/indexer/test_cli.py [hunk 2]
```diff
@@ -192,3 +192,82 @@ def test_remap_index_when_there_are_more_than_one_indexes_with_same_name_but_dif
...
+    assert "test-alias-prefix-records-authors" in aliases
+    assert "test-alias-prefix-records-seminars" in aliases
+    assert "test-alias-prefix-records-jobs" in aliases
+    assert "test-alias-prefix-records-institutions" in aliases
+    assert "test-alias-prefix-records-journals" in aliases
+    assert "test-alias-prefix-records-data" in aliases
+    assert "test-alias-prefix-records-experiments" in aliases
+    assert "test-alias-prefix-records-conferences" in aliases
+
+    # cleanup
```
Comment (drjova) [inline backend/tests/integration/indexer/test_cli.py:271]: redundant

File: backend/tests/integration/indexer/test_cli.py [hunk 3]
```diff
@@ -192,3 +192,82 @@ def test_remap_index_when_there_are_more_than_one_indexes_with_same_name_but_dif
...
+
+    assert "records-hep" in aliases
+    assert "records-authors" in aliases
+    assert "records-seminars" in aliases
+    assert "records-jobs" in aliases
+    assert "records-institutions" in aliases
+    assert "records-journals" in aliases
+    assert "records-data" in aliases
+    assert "records-experiments" in aliases
+    assert "records-conferences" in aliases
```
Comment (drjova) [inline backend/tests/integration/indexer/test_cli.py:221]: could we also test where they are pointing?

File: backend/tests/integration/indexer/test_cli.py (hunk 1 above)
Comment (pazembrz) [reply backend/tests/integration/indexer/test_cli.py:223]: Why? They are separate aliases.

File: backend/tests/integration/indexer/test_cli.py (hunk 2 above)
Comment (pazembrz) [reply backend/tests/integration/indexer/test_cli.py:271]: also here

File: backend/tests/integration/indexer/test_cli.py (hunk 1 above)
Comment (drjova) [reply backend/tests/integration/indexer/test_cli.py:223]: the comment :)

Comment (pazembrz) [reply backend/tests/integration/indexer/test_cli.py:223]: :+1:

File: backend/tests/integration/indexer/test_cli.py (hunk 2 above)
Comment (pazembrz) [reply backend/tests/integration/indexer/test_cli.py:271]: :+1:

File: backend/tests/integration/indexer/test_cli.py (hunk 3 above)
Comment (pazembrz) [reply backend/tests/integration/indexer/test_cli.py:229]: :+1:

**Pull request merged**